packages/edit-navigation/src/components/menu-editor/navigation-structure-panel.js

```diff
@@ -26,6 +26,7 @@ export default function NavigationStructurePanel( { blocks, initialOpen } ) {
 						selectedBlockClientId={ selectedBlockClientIds[ 0 ] }
 						selectBlock={ selectBlock }
 						__experimentalWithBlockNavigationSlots={ true }
+						__experimentalWithEllipsisMenu={ true }
```

Comment:
Could omit the `true`.

```suggestion
						__experimentalWithBlockNavigationSlots
						__experimentalWithEllipsisMenu
```